tests/probes/process58/test_probes_process58_offline_mode.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+. $builddir/tests/test_common.sh
+
+set -e -o pipefail
+
+function test_probes_process58_offline_mode {
+    probecheck "process58" || return 255
+
+    local ret_val=0;
+    local DF="${srcdir}/test_probes_process58_offline_mode.xml"
+    local RF="test_probes_process58_offline_mode.results.xml"
+    echo "result file: $RF"
+    local stderr=$(mktemp $1.err.XXXXXX)
+    echo "stderr file: $stderr"
+
+    [ -f $RF ] && rm -f $RF
+
+    tmpdir=$(mktemp -t -d "test_offline_mode_process58.XXXXXX")
+
+    ln -s /proc $tmpdir
```

Comment:
Same here and probably also in other offline mode tests - maybe we can test empty filesystem which emulates missing /proc directory, probe should be able to handle it